app/helpers/markup_helper.rb

```diff
@@ -0,0 +1,35 @@
+module MarkupHelper
```

Comment:
Nothing wrong with this that I can see off the top of my head, but why not just use an existing Markdown library?